cfmm.mligo

```diff
@@ -402,14 +481,23 @@ let remove_liquidity (param : remove_liquidity) (storage : storage) : result =
         end
     end
 
+let ctez_target (param : ctez_target) (storage : storage) = 
```

Comment:
Where does ctez.mligo call ctez_target?

Reply:
Just added so that each time `ctez.mligo` updates its target locally, it feeds it to the cfmm